lib/sendAnonymousCLIMetrics.js

```diff
@@ -0,0 +1,19 @@
+module.exports = send
+
+var assert = require('assert')
+var url = require('url')
+
+function send (uri, params, cb) {
```

Comment:
This function should compose its URL on its own (for example how `lib/dist-tags/set.js` works).